user_gen.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+
+
+def poisson_process(rate=1, size=None):
+    exp_draws = np.random.exponential(scale=rate, size=size)
+    sum_exp_draws = [sum(exp_draws[:i]) for i, x in enumerate(exp_draws)]
+    return sum_exp_draws
+
+
+def poisson_plus(lam=1, plus=1, size=None):
+    return np.random.poisson(lam=lam, size=size) + plus
+
+
+def int_exponential(rate=1, size=None):
+    exp_draws = np.random.exponential(scale=rate, size=size)
+    return np.round(exp_draws)
+
+
+def is_pos_inc_seq(seq):
+    return (sorted(seq) == list(seq) and all([x >= 0 for x in seq]))
+
+
+def is_any_int(x):
+    return (isinstance(x, int)
+            or isinstance(x, np.int64))
+
+
+def is_pos_int(seq):
+    return (all([(x > 0 and is_any_int(x)) for x in seq]))
+
+
+class Simple_Job_Set_Manager(object):
+    def __init__(self, num_jobs):
+        self.num_jobs = num_jobs
+
+    @property
+    def users_per_job(self):
+        return self._users_per_job
```

Comment:
I try to keep `__init__` decently bear since otherwise the calls will be terribly unwieldy, especially if we're passing functions and parameters around. 

Additionally, I want people to be able to assign stuff after the fact & we need to have error checking on those assignments for us not to run into errors further down the road.